contract/package.json

```diff
@@ -7,10 +7,13 @@
   "scripts": {
     "start:docker": "docker compose up -d",
     "docker:logs": "docker compose logs --tail 200 -f",
-    "docker:bash": "docker compose exec agd bash",
-    "docker:make": "docker compose exec agd make -C /workspace/contract",
+    "docker:bash": "docker exec agd bash",
+    "docker:make": "docker exec agd make -C /ws-agoric-basics/contract",
```

Comment:
interesting approach!

Reply:
idea: use an env variable with a default `${AG_CONTAINER:-agd}`

and maybe for `ws-agoric-basics`